lemur/tests/test_integration_utility_generate_and_convert.py

```diff
@@ -1,243 +0,0 @@
-# This file consists of testing for functions that generate messages/strings
```

Comment:
i wouldn't delete this file-- you want both the integration and unit tests. Best practice would be to set your testing command up so it runs the unit tests by default, and runs unit and integration tests if you specify a special parameter. Then you'd set up a continuous integration server to run all tests (integration and unit) whenever a new PR is submitted.